head.go

```diff
@@ -812,6 +825,17 @@ func (s *memSeries) append(t int64, v float64) bool {
 	return true
 }
 
+// computeEndTime estimates the end timestamp based the beginning of a chunk,
+// its current timestamp and the upper bound up to which we insert data.
+// It assumes that the time range is 1/4 full.
+func computeChunkEndTime(start, cur, max int64) int64 {
+	a := (max - start) / (cur - start + 1) / 4
```

Comment:
Umm, maybe its just me, but can we have it as:
```
a := (max - start) / ((cur - start + 1) * 4)
```

Left me a little confused and made me resort to play.golang.org